android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/CloudRepository.java

```diff
@@ -646,6 +665,7 @@ protected void onPostExecute(CloudDownloadReturns jsonTuple) {
     }
 
     public void processCloudReturns(CloudDownloadReturns jsonTuple, boolean executeCallbacks) {
+      // TODO:  Need proper offline check again.
```

Comment:
I should probably make an issue here, just in case.

Reply:
👍

Reply:
Now handled by an update in #1845.